google/cloud/spanner_dbapi/connection.py

```diff
@@ -190,6 +265,33 @@ def run_prior_DDL_statements(self):
 
             return self.database.update_ddl(ddl_statements).result()
 
+    def run_statement(self, statement, retried=False):
+        """Run single SQL statement in begun transaction.
+
+        This method is never used in autocommit mode. In
+        !autocommit mode however it remembers every executed
+        SQL statement with its parameters.
+
+        :type statement: :class:`dict`
+        :param statement: SQL statement to execute.
+
+        :rtype: :class:`google.cloud.spanner_v1.streamed.StreamedResultSet`,
+                :class:`google.cloud.spanner_dbapi.checksum.ResultsChecksum`
+        :returns: Streamed result set of the statement and a
+                  checksum of this statement results.
+        """
+        transaction = self.transaction_checkout()
+        self._statements.append(statement)
+
+        return (
+            transaction.execute_sql(
```

Comment:
Is this method using the `executeStreamingSql` RPC? Or is it using the `executeSql` RPC?

Reply:
This method is inherited from `_SnapshotBase` class, here it is:
https://github.com/googleapis/python-spanner/blob/783fffb6683119db0dde89bff66112e451881c19/google/cloud/spanner_v1/snapshot.py#L178-L189

So, it's `ExecuteStreamingSql`.